packages/rs-drive-abci/src/execution/validation/state_transition/check_tx_verification/v0/mod.rs

```diff
@@ -65,6 +64,32 @@ pub(super) fn state_transition_to_execution_event_for_check_tx_v0<'a, C: CoreRPC
                 }
             }
 
+            // Disable contested document create transitions for the first 2 epochs
+            // We doing it very top of state transition validation logic to avoid any unnecessary expenses
+            // for a state transition owner.
+            let result = state_transition.validate_temporary_disabled_contested_documents(
+                platform.state.last_block_info(),
+                platform_version,
+            )?;
```

Comment:
I considered this solution. I see two downsides:
1. It's a bad architecture since basic structure validation is implemented in DPP and not supposed to validate against state or any other context. It only checks state transition structure. Passing block info there will harm this core architecture concept. 
2. A user still will pay fees for nonce and signature (process state transition) validation